.changeset/eight-dots-eat.md

```diff
@@ -0,0 +1,5 @@
+---
+"@infra/resources": minor
```

Comment:
**question**: minor or patch?

Reply:
Updated to patch in acbf3bec67503f6c37e36a4fa82d031c2b8e84ec